<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLJobWindow</class>
 <widget class="QMainWindow" name="FLJobWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLJobWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QDockWidget" name="dockWidget">
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="windowTitle">
       <string>Jobs</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents"/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDockWidget" name="dockWidget_3">
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Properties</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionWrite"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/open-icon.png</normaloff>:/firelink/Resources/icons/open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/add-folder-icon.png</normaloff>:/firelink/Resources/icons/add-folder-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/save-icon.png</normaloff>:/firelink/Resources/icons/save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/remove-folder-icon.png</normaloff>:/firelink/Resources/icons/remove-folder-icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>close job</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/firmware_update.png</normaloff>:/firelink/Resources/icons/firmware_update.png</iconset>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionDockProperty">
   <property name="icon">
    <iconset resource="firelink.qrc">
     <normaloff>:/firelink/Resources/icons/pin-icon.png</normaloff>:/firelink/Resources/icons/pin-icon.png</iconset>
   </property>
   <property name="text">
    <string>DockProperty</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="firelink.qrc"/>
 </resources>
 <connections/>
</ui>
